client/tests/integration/triggers/by_call_trigger.rs

```diff
@@ -357,7 +357,7 @@ fn trigger_in_genesis_using_base64() -> Result<()> {
 
     // Registering trigger in genesis
     let mut genesis = GenesisNetwork::test(true).expect("Expected genesis");
-    match &mut genesis.transactions[0].as_mut_v1().payload.instructions {
+    match &mut genesis.transactions[0].as_mut_v1().0.payload.instructions {
```

Comment:
> I see many places where we use this as_mut_v1().0.payload boilerplate and also know that we have an immutable method to retrieve payload

we shouldn't be having mutable getters but I didn't want to go into handling this since we're moving to python based integration tests. Getters are there only to opacify our data model structs and those structs are immutable from the perspective of the end user

Reply:
Got you